istioctl/cmd/kubeinject.go

```diff
@@ -393,8 +393,9 @@ func setupKubeInjectParameters(sidecarTemplate *inject.RawTemplates, valuesConfi
 			if *sidecarTemplate, err = getInjectConfigFromConfigMap(kubeconfig, revision); err != nil {
 				return nil, nil, err
 			}
+		} else {
+			return injector, nil, nil
```

Comment:
Could we improve error msg in this case?  something to let user know the mutating web hook config can't be found and alert user to use --revision if needed.

Reply:
The lack of a revision flag is not the only reason why the fallback may be taken. It does so on any error so the message would have to be more generic.